src/alg_utils.jl

```diff
@@ -321,6 +322,7 @@ beta1_default(alg::DP5,beta2) = typeof(beta2)(1//alg_order(alg)) - 3beta2/4
 beta1_default(alg::DP5Threaded,beta2) = typeof(beta2)(1//alg_order(alg)) - 3beta2/4
 
 gamma_default(alg::OrdinaryDiffEqAlgorithm) = 9//10
+gamma_default(alg::RKC) = 8//10
```

Comment:
@ChrisRackauckas @YingboMa Changed gamma to match the constants of the paper

Reply:
yeah that's fine